src/lib/repoStatusCode.js

```diff
@@ -0,0 +1,5 @@
+export async function repoStatusCode(repoUrl) {
+  const apiRepoUrl = `https://api.github.com/repos/${repoUrl}`;
+  const response = await fetch(apiRepoUrl);
+  return response.status;
+}
```

Comment:
This is exactly what I was looking for. Thanks for working on this and writing a test. 

We could do something similar for the duplicate issues. Validate if they exist already in the goals repo. https://github.com/open-sauced/open-sauced/issues/555

Reply:
> We could do something similar for the duplicate issues. Validate if they exist already in the goals repo. #555

@bdougie Do we need to handle in this PR or in separate ones?

Reply:
Depends on you, the context of the PR fits here. It would need another validation for issues using the title (owner/repo) to search the issue exists (open or closed).

Reply:
can you check to see if the response was redirected? 

`response.redirected`

https://javascript.info/fetch-api#redirect